src/javascript/app/pages/user/new_account/real_acc_opening.js

```diff
@@ -13,12 +13,15 @@ const RealAccOpening = (() => {
         if (Client.get('residence')) {
             if (AccountOpening.redirectAccount()) return;
 
-            BinarySocket.wait('landing_company').then(() => {
+            BinarySocket.wait('authorize', 'landing_company').then(() => {
```

Comment:
this page is already an authorized page, meaning we already wait for authorize before loading this page, handled in binary_loader.js. you don't need this additional wait for authorize.